configs/templates/_gce.txt

```diff
@@ -28,7 +28,7 @@ SSH_KEY_NAME = $GCE_SSH_KEY_NAME
 
 [MON_DEFAULTS : GCE_CLOUDCONFIG ]
 COLLECT_FROM_HOST = $False # impossible for amazon
-COLLECT_FROM_GUEST = $True
+COLLECT_FROM_GUEST = $False
```

Comment:
I was trying to be a minimalist here, and just follow the procedure adopted on all other adapters (please note that this - COLLECT_FROM_GUEST = $False - is the case for all cloud adapters). Given that deployments might fail due to errors on the restarting of gmond, leave this option disabled by default. The goal here is to have an AI up and running as fast as possible, without any errors, and only then activate more "features" (such as Guest OS resource metrics) gradually.